README.md

```diff
@@ -7,38 +7,21 @@
+In few words, the app state is entirely described by a single serializable data structure, and the only way to change the state is to dispatch an action. An action is an intent to transform the state, and contains all the information to do so. Because all the changes are centralized and are happening in a strict order, there are no subtle race conditions to watch out for.
 
-- __logic__: the app state is entirely described by a single serializable data structure, and the only way to change the state is to dispatch an action. An action is an intent to transform the state, and contains all the information to do so. Because all the changes are centralized and are happening in a strict order, there are no subtle race conditions to watch out for.
-- __UI__: the UI is defined in terms of a tree of components declaratively described by props (the configuration data, i.e. a background color for a button) and state (the internal state data, i.e. the highlighted state for a button). This approach lets you think about components as isolated, reusable pieces of UI, since the way a component is rendered only depends on the current props and state of the component itself.
-- __logic__ ↔️ __UI__: the UI components are connected to the app state and will be automatically updated on every state change. You control how they change, selecting the portion of app state that will feed the component props. To render this process as fast as possible, only the relevant portion of the UI is updated. 
-- __layout__: Katana defines a concise language (inspired by [Plastic](https://github.com/BendingSpoons/plastic-lib-iOS)) to describe fully responsive layouts that will gracefully scale at every aspect ratio or size, including font sizes and images.
-
-
-We feel that Katana helped us a lot since we started using it in production. At [Bending Spoons](http://www.bendingspoons.com) we use a lot of open source projects ourselves and we wanted to give something back to the community, hoping you will find this useful and possibly contribute. ❤️ 
-
-
-
-|      | Katana                                   |
-| ---- | ---------------------------------------- |
-| 🎙   | Declaratively define your UI             |
-| 📦   | Store all your app state in a single place |
-| 💂   | Clearly define what are the actions that can change the state |
-| 😎   | Describe asynchronous actions like HTTP requests |
-| 💪   | Support for middleware                   |
-| 🎩   | Automatically update the UI when your app state changes |
-| 📐   | Automatically scale your UI to every size and aspect ratio |
-| 🐎   | Easily animate UI changes                |
-| 📝   | Gradually migrate your application to Katana |
+We feel that Katana helped us a lot since we started using it in production.  At [Bending Spoons](http://www.bendingspoons.com) we use a lot of open source projects ourselves and we wanted to give something back to the community, hoping you will find this useful and possibly contribute. ❤️ 
```

Comment:
`production` -> why don't we stress the fact that katana is behind apps used by millions of users?